src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -226,10 +226,19 @@ public void PushWithTransition(Page page)
             transitionSet = CreateTransitions(topPage, page, true);
             transitionSet.Finished += (object sender, EventArgs e) =>
             {
+                HideContentOfPage(topPage);
```

Comment:
Is this required here?
Because it seems that Push() works fine without this code.

Reply:
Ah
Thanks for checking it. I didn't delete all the old codes I've tested.
Remove it and make the same as Push().